src/UnitySampleProject/Assets/Scripts/Scenes/WebLogin.cs

```diff
@@ -22,9 +22,9 @@ public class WebLogin : MonoBehaviour
     void Start()
     {
         // loads the data saved from the editor config
-        projectConfigSO = (ProjectConfigScriptableObject)Resources.Load("ProjectConfigData", typeof(ScriptableObject));
-        PlayerPrefs.SetString("ProjectID", projectConfigSO.ProjectID);
-        PlayerPrefs.SetString("ChainID", projectConfigSO.ChainID);
+        projectConfigSO = ProjectConfigScriptableObject.LoadDefault();
```

Comment:
This has to be `ProjectConfigUtilities.Load()`. I think you made the same mistake as me: not having WebGL as the active platform when working on the codebase.